src/utils/DateUtils.ts

```diff
@@ -24,6 +25,13 @@ class DateUtils {
       DateUtils.DATE_FORMAT_OPTIONS
     )
   }
+
+  static getCredentialStatus(expirationDate: number): CredentialStatusEnum {
```

Comment:
would move this to a src/utils/CredentialUtils

Reply:
Moved to CredentialUtils.
No need to make the interface we are determining the status of the credential. The credential is now used as param.